tests/rgb_tests.cpp

```diff
@@ -34,6 +35,12 @@ constexpr uint8_t rgb6_to_8_reference(const uint8_t c)
 
 constexpr auto AbsError = 0.000001f;
 
+TEST(rgb, Rgb888_FromRgb444)
+{
+	EXPECT_EQ(Rgb888().FromRgb444(0x0, 0x1, 0x2), Rgb888(0x00, 0x11, 0x22));
+	EXPECT_EQ(Rgb888().FromRgb444(0x8, 0xe, 0xf), Rgb888(0x88, 0xee, 0xff));
```

Comment:
Static class functions can be called more like a scoped function or an enum class function (if those existed):

```Rgb888::FromRgb444(args)```